src/gwc/src/main/java/org/geoserver/gwc/GWC.java

```diff
@@ -672,6 +675,29 @@ public final ConveyorTile dispatch(final GetMapRequest request,
             return null;
         }
 
+        if (getConfig().isSecurityEnabled()) {
+            String bboxstr = request.getRawKvp().get("BBOX");
+            String srs = request.getRawKvp().get("SRS");
+            ReferencedEnvelope bbox = null;
+            try {
+                bbox = (ReferencedEnvelope) new BBoxKvpParser().parse(bboxstr);
+            } catch (Exception e) {
+                throw new RuntimeException("Invalid bbox for layer '" + layerName + "': " + bboxstr);
+            }
+            if (srs != null) {
+                try {
+                    bbox = new ReferencedEnvelope(bbox, CRS.decode(srs));
```

Comment:
GWC doesn't do the magic version dependant swapping for 1.3.0, which would seem to be a bug in its own right under integrated mode which advertises 1.3.0 but then delegates to GWC, but it means that this incorrect behaviour balances out the incorrect behaviour elsewhere and masks the areas we want masked.

I REALLY hate these axis order bugs.